src/ProxyClient/MultiplexerClient.php

```diff
@@ -96,6 +98,38 @@ public function flush()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getTagsHeaderValue(array $tags)
+    {
+        return $this->invoke(TagCapable::class, 'getTagsHeaderValue', [$tags]);
```

Comment:
this is not supposed to change anything on the client, so i think we should instead just call the first client and return that, instead of calling all of them and then return the last.